ci/requirements/environment.yml

```
@@ -25,7 +25,7 @@ dependencies:
   - netcdf4
   - numba
   - numexpr
-  - numpy
+  - numpy=1.20
```

```suggestion
  - numpy
```
Remove this before merge.